src/pmpo_wachspressBasis.hpp

```diff
@@ -127,7 +127,27 @@ void getBasisByAreaGblForm(Vec2d MP, int numVtxs, Vec2d* vtxCoords, double* basi
     }
 
     calcBasis(numVtxs, a, c, basis);
-}// getBasisByAreaBblForm
+}// getBasisByAreaGblForm
```

Comment:
Please remove these comments (i.e., `// getBasisByAreaGblForm`) on the closing brackets of functions.  If our functions don't fit on a single 'screen' of the editor that can indicate problems.  Also, Vim, VSCode and other editors have ways of showing you the matching brackets.